docs/config.md

```diff
@@ -8,7 +8,7 @@ These are the config settings available to jj/Jujutsu.
 The config settings are loaded from the following locations. Less common ways to
 specify `jj` config settings are discussed in a later section.
 
-* `~/.jjconfig.toml` (global)
+* `~/.config/jj/config.toml` (global)
```

Comment:
I'm worried the `~/.config/jj/config.toml` path might not actually be correct on Windows. I thing it uses some sort of a `%USER_DATA_SOMETHING_SOMETHING%` variable.

Reply:
Yes, that's why I changed (back) to `~/.jjconfig.toml` in commit 96849da332c809041040371b55eb5b4aaf097330. Now that we have this document, maybe we can describe the different locations here and then just try to say "in your config file" or something elsewhere. I'm not sure if that's better.

Reply:
It might be nice to make this clearer in config.md and the comments of config.rs.

Reply:
Reworded things to make config.md the single source of truth.